source/agora/consensus/data/genesis/Test.d

```diff
@@ -51,6 +51,7 @@ public immutable Block GenesisBlock = {
         prev_block:  Hash.init,
         height:      Height(0),
         merkle_root: GenesisMerkleTree[$ - 1],
+        timestamp:   1596153600,
```

Comment:
Won't that cause problems in tests, since it's set to 2020-07-31 (also please add a comment about it) ?

Reply:
that is a good point and it took me days to find out why tests fails randomly when running in multi-threaded mode(nomination was kicking out too early)

last week I solved that by adding an overriden version of the 
protected override ulong getExpectedBlockTime () @safe @nogc nothrow pure
method to the TestNominator